aws_cloudtrail_rules/aws_console_login_without_mfa.py

```diff
@@ -10,6 +14,28 @@ def rule(event):
     session_context = deep_get(event, "userIdentity", "sessionContext", default={})
     response_elements = event.get("responseElements", {})
 
+    # If Account is less than 3 days old do not alert
+    # This functionality is not enabled by default, in order to start logging new user creations
+    # Enable indicator_creation_rules/new_account_logging to start logging new users
+    new_user_string = (
+        deep_get(event, "userIdentity", "userName", default="<MISSING_USER_NAME>")
+        + "-"
+        + deep_get(event, "userIdentity", "principalId", default="<MISSING_ID>")
+    )
+    is_new_user = check_account_age(new_user_string)
```

Comment:
Since the helper is disabled, this will always trigger, right? Basically keeping the original logic and folks can optionally turn on the new user rule if they so choose to "enable" this logic? (just making sure I follow :) )